packages/flutter_tools/lib/src/android/gradle_errors.dart

```diff
@@ -120,6 +126,15 @@ final GradleHandledError networkErrorHandler = GradleHandledError(
       '$warningMark Gradle threw an error while downloading artifacts from the network. '
       'Retrying to download...'
     );
+    try {
+      final String homeDir = globals.platform.environment['HOME'];
```

Comment:
Wouldn't this be in %LOCALAPPDATA% or whatever on Windows? Can we try to check there?

Reply:
Its in HOME for me

Reply:
Ahh, its even documented: https://docs.gradle.org/current/userguide/directory_layout.html#dir:gradle_user_home